scully.log

```diff
@@ -0,0 +1,222 @@
+No routes defined in ""The option outDir isn't configured, using default folder "[33m.\dist\static\[39m".
+
+No routes defined in ""The option outDir isn't configured, using default folder "[33m.\dist\static\[39m".
```

Comment:
this file should not be committed. Here is what needs to be done:

1. delete this file and commit the changes
2. add `scully.log` to `.gitignore` and commit again

Reply:
added